### ISSUE TITLE
Old Submitted dataset just now appearing in Submitted queue

### ISSUE THREAD

**Issue by tracykteal** [CDL-Dryad/dryad-product-roadmap#1127]
**Bug description:**
A couple of datasets are just appearing in the Submitted queue from March and October that we haven't seen in the queue previously, and logs don't show that anything has changed recently.
- 10.5061/dryad.tx95x69sw
- 10.5061/dryad.bnzs7h470

Here's a screenshot. Selecting 'Submitted' as the Status and then sorting by oldest date first.

![Screen Shot 2021-02-18 at 10 01 55 AM](https://user-images.githubusercontent.com/889238/108401056-c825f880-71d0-11eb-90e2-346a8f2bb108.png)

**Comment by sfisher** [comment]
I had a quick look at these items.

We keep timestamps for all database changes and there have been no recent changes on these.  The submissions logs are from the original time frame, the status changes are from the original times of submission (multiple of them).  The queue states are from the original times.  Nothing seems to have been changed with regard to these items anytime recently and we track change times in multiple tables around submissions and different events.

The code for the curation status screen hasn't had any changes I'm aware of recently, so I see no reason these would suddenly appear.

Is it possible that they were missed because this exact combination of status: `submitted` and sort order `last modified: ascending` (A to Z icon) wasn't used or additional filters weren't in place?

I just don't see any indication that anything has changed that would have changed this anytime recently, though I do notice that both were in "peer review" in earlier versions of the datasets.  Perhaps it's something we can look at as a possible cause, but I don't think we've particularly seen peer review problems that I'm aware of.

Further information for our internal troubleshooting:  identifier id: 37215, latest resource: 88071,    identifier_id: 43554, resource: 55578 .

**Comment by tracykteal** [comment]
Thanks for looking into this. This is a regular filter and sort that we use, so I don't think it's something new there. We'll keep an eye on it to see if we see it again and will report if we do.

**Comment by ryscher** [comment]
I pulled an old databases backup and looked at these items in more detail. 

In both cases, shortly after a submission, the user created a new version and
that version remained `in progress`. This explains why the items did not appear
in the curation queue. At some point between February 15th and February 20th,
the `in progress` versions were removed, and the items reverted to their
previous `submitted` state, so they started appearing in the curation
queue.

It's likely that I was the person who removed them. Although I have no memory of these particular items, and I cannot find record of discussing them, I do remember cleaning up some broken items around that time. I will be more careful about leaving provenance notes in the future when I work with items that are in broken states.
